src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -26,6 +36,11 @@ public class ShooterConstants {
     public static final InvertedValue TOP_INVERSION = InvertedValue.Clockwise_Positive;
     public static final InvertedValue BOTTOM_INVERSION = InvertedValue.CounterClockwise_Positive;
 
+    public static final InterpolatingDoubleMap VELOCITY_BY_DISTANCE =
+            ShootingCSV.parse("distance-to-velocity.csv"); // Velocity | Distance
+    public static final InterpolatingDoubleMap FLIGHT_TIME_BY_DISTANCE =
+            ShootingCSV.parse("distance-to-flight-time.csv"); // Flight Time | Distance
```

Comment:
The paths should be relative to the deploy directory, otherwise the code won't find the files.

Reply:
a65809b6029a62dd1548fe640415b1c1bbdb9f77

Reply:
These paths will be relative to whatever the working directory is on the roboRIO. WPILib has a handy function that will help you.

Reply:
aa4fa06

Reply:
Apparently that's not the correct path

Reply:
fa681a3 fixed path
But there is still an error with reading the file